### PR TITLE
Reword "Content Management Guide" to "Managing Content"

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/ref_initialization-script-for-provisioning-examples.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ NOTE: The following chapter describes Red Hat content management provided by Kat
 Deployments without the plugin use Installation Media to fetch content from remote or local mirrors.
 endif::[]
 
-If you have not followed the examples in the {ProjectName} Content Management Guide, you can use the following initialization script to create an environment for provisioning examples.
+If you have not followed the examples in {ContentManagementDocTitle}, you can use the following initialization script to create an environment for provisioning examples.
```
Comment (adamlazik1) [inline guides/common/modules/ref_initialization-script-for-provisioning-examples.adoc:9]:
```suggestion
If you have not followed the examples in _{ContentManagementDocTitle}_, you can use the following initialization script to create an environment for provisioning examples.
```

File: guides/doc-Planning_for_Project/topics/Introduction.adoc [hunk 2]
```diff
@@ -100,7 +100,7 @@ The new Content View version is then promoted through life cycle environments.
 You can also create in-place updates of Content Views.
 This means creating a minor version of the Content View in its current life cycle environment without promoting it from the Library.
 For example, if you need to apply a security erratum to a Content View used in Production, you can update the Content View directly without promoting to other life cycles.
-For more information on content management see the {ContentManagementDocURL}[Content Management Guide].
+For more information on content management see the {ContentManagementDocURL}[{ContentManagementDocTitle}].
```
Comment (adamlazik1) [inline guides/doc-Planning_for_Project/topics/Introduction.adoc:103]:
```suggestion
For more information on content management, see {ContentManagementDocURL}[{ContentManagementDocTitle}].
```

**Review by maximiliankolb — COMMENTED** [review]
Thanks. Applied both suggestions and rebased to master. Ready for re-review.

File: guides/doc-Planning_for_Project/topics/Introduction.adoc [hunk 3]
```diff
@@ -100,7 +100,7 @@ The new Content View version is then promoted through life cycle environments.
 You can also create in-place updates of Content Views.
 This means creating a minor version of the Content View in its current life cycle environment without promoting it from the Library.
 For example, if you need to apply a security erratum to a Content View used in Production, you can update the Content View directly without promoting to other life cycles.
-For more information on content management see the {ContentManagementDocURL}[Content Management Guide].
+For more information on content management, see {ContentManagementDocURL}[_{ContentManagementDocTitle}_].
```
Comment (adamlazik1) [inline guides/doc-Planning_for_Project/topics/Introduction.adoc:103]: I don't know if italic should be used on title with embedded link. I did notice not this in the docs.

**Comment by maximiliankolb** [comment]
Thanks @adamlazik1 ready for re-review.

**Review by adamlazik1 — COMMENTED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked the second commit: ✅ 
   1fc42c048..d7046e038  3.4 -> 3.4
   9d0eb503b..b94e9580c  3.3 -> 3.3
   e6e5e39ba..9691c82d5  3.2 -> 3.2
   39827d580..ae742cfba  3.1 -> 3.1

**Pull request merged**